### PR TITLE
two new asynch methods

### PR DESCRIPTION
**Pull request by victorsosa**
Hi,
Here you go two new asynch methods. 
I am using it for the project nativescript-svg.

imageWithContentsOfFileAsynchronously(path, callback)
imageWithDataAsynchronously(data, callback)

### CODE REVIEW

File: Source/SVGKImage.h [hunk 1]
```diff
@@ -100,7 +100,7 @@ typedef void (^SVGKImageAsynchronousLoadingDelegate)(SVGKImage* loadedImage, SVG
  */
 +(SVGKParser *) imageAsynchronouslyNamed:(NSString *)name onCompletion:(SVGKImageAsynchronousLoadingDelegate) blockCompleted;
 + (SVGKImage *)imageWithContentsOfFile:(NSString *)path;
-
++ (SVGKImage*) imageWithContentsOfFileAsynchronously:(NSString *)aPath onCompletion:(SVGKImageAsynchronousLoadingDelegate)blockCompleted
```
Comment (Harry-L) [inline Source/SVGKImage.h:103]: Missing semicolon

Comment (victorsosa) [inline Source/SVGKImage.h:103]: Thanks, I am fixing it.

Comment (victorsosa) [inline Source/SVGKImage.h:103]: PR #451 sent

File: Source/SVGKImage.m [hunk 2]
```diff
@@ -222,6 +226,14 @@ + (SVGKImage*) imageWithData:(NSData *)newNSData
     }
 }
 
++ (SVGKImage*) imageWithDataAsynchronously:(NSData *)newNSData onCompletion:(SVGKImageAsynchronousLoadingDelegate)blockCompleted
+{
+	NSParameterAssert(newNSData != nil);
+	SVGKitLogWarn(@"Creating an SVG from raw data; this is not recommended: SVG requires knowledge of at least the URL where it came from (as it can contain relative file-links internally). You should use the method [SVGKImage initWithSource:] instead and specify an SVGKSource with more detail" );
+
+	return [self imageWithSource:[SVGKSourceNSData sourceFromData:newNSData] onCompletion:blockCompleted];
```
Comment (Harry-L) [inline Source/SVGKImage.m:234]: missing URLForRelativeLinks argument I think

Comment (victorsosa) [inline Source/SVGKImage.m:234]: NOP, I am using imageWithSource method no the initWithSource ;) thanks for double check

Comment (victorsosa) [inline Source/SVGKImage.m]: the comment line can be removed

Comment (Harry-L) [inline Source/SVGKImage.m:234]: I'm referring to the SVGKSourceNSData initialization not the imageWithSource method.

Comment (victorsosa) [inline Source/SVGKImage.m:234]: yep, you are right,

**Pull request merged**